src/cmd/usb-util.js

```diff
@@ -183,6 +182,10 @@ async function waitForDeviceToRespond(deviceId, { timeout = 10000 } = {}) {
 			return device;
 		} catch (error) {
 			// ignore errors
+			// device could be open after the last iteration
+			if (device && device.isOpen) {
+				await device.close();
+			}
 		}
```

Comment:
This is not a part of the PR description. I needed this because I was noticing that if the device never responds to a control request, for example, the UI is hanging there. The UI should return to the next line.